### PR TITLE
Adjusts scale and angular range in 1D plots from WedgeSlicer

### PR DESCRIPTION
**Pull request by butlerpd**
## Description

The 1D plots created by the new `WedgeSlicer` now returns:
* Properly converted radians to degrees for the x scale
* The default scale for both x (phi) and y (intensity) are linear.
* The Angular range is pi to pi (with 0 pointing along the positive x axis) as per the normal python and sasview convention

NOTE: this was tested and works well with uncommitted changes to manipulation.py in sasdata needed to fix the actual averaging. This can still be tested with sasdata/master I think as long as one does not allow the wedge to straddle the negative x axis. This should not affect merging since that problem exists in the current main of sasview.

NOTE2:  The caveat to the angular range is that when the wedge straddles the negative x axis the phi max will be greater than 2pi but the data will be in proper consecutive order. This is mostly coming from sasdata. While this may seem a bit odd it is the only rational way to display such data.

Fixes #2578

## How Has This Been Tested?

run on windows 10 developer and checked the wedge slicer (moving the different interactors around), and compared to the annular and sector interactors

## Review Checklist (please remove items if they don't apply):

- [ ] Code has been reviewed
- [ ] Functionality has been tested
- [ ] Windows installer (GH artifact) has been tested (installed and worked) 
- [ ] MacOSX installer (GH artifact) has been tested (installed and worked) 
- [ ] User documentation is available and complete (if required)
- [ ] Developers documentation is available and complete (if required)
- [ ] The introduced changes comply with SasView license (BSD 3-Clause)

### CODE REVIEW

File: src/sas/qtgui/Plotting/Slicers/WedgeSlicer.py [hunk 1]
```diff
@@ -175,7 +176,10 @@ def _post_data(self, new_sector=None, nbins=None):
             dxw = sector.dxw
         else:
             dxw = None
-        new_plot = Data1D(x=sector.x, y=sector.y, dy=sector.dy, dx=sector.dx)
+        # And here subtract pi when getting angular data back from sector averaging in
+        # manipulations to get back in the -pi,pi range. Also convert from radians to
+        # degrees for nicer display.
+        new_plot = Data1D(x=(sector.x - np.pi) * 180 / np.pi, y=sector.y, dy=sector.dy, dx=sector.dx)
```
Comment (ehewins) [inline src/sas/qtgui/Plotting/Slicers/WedgeSlicer.py:182]: At the moment, this angular shift is happening even when plotting against Q. I think this line should go back to having `x=sector.x`, and then the following could be added after line 169:
```python
        if self.averager.__name__ == 'SectorPhi':
            sector.x = (sector.x - np.pi) * 180 / np.pi
```

Comment (butlerpd) [reply src/sas/qtgui/Plotting/Slicers/WedgeSlicer.py:182]: Good catch @ehewins

Comment (butlerpd) [reply src/sas/qtgui/Plotting/Slicers/WedgeSlicer.py]: In principle if class.__name__ ==name: is not a good construct I've been told. However there are 2 other uses of it in this one method so just going with it for this PR but left a "todo" comment. I suspect the right answer would be a create_plot method in post_data that handles those 3 specifics?

**Comment by butlerpd** [comment]
As pointed out by @ehewins the slicers seem to be rife with the use of  `self.classname.__name__` which maybe needs to be looked at more carefully. In the meantime agreed to merge this.

**Pull request merged**